doc/visual-programming/source/loading-your-data/index.rst

```diff
@@ -6,19 +6,18 @@ also handle native Excel (.xlsx or .xls), comma- or tab-delimited data files. Th
 set is usually a table, with data instances (samples) in rows and
 data attributes in columns. Attributes can be of different type
 (continuous, discrete, and strings) and kind (input features, meta attributes, and class). Data attribute type and kind can be provided
-in the data table header. This can be changed later, after reading the
-data with :doc:`Select Columns<../widgets/data/selectcolumns>` widget.
+in the data table header. They can also be subsequently changed in the :doc:`File<../widgets/data/file>` widget, 
+while data kind can also be modified with :doc:`Select Columns<../widgets/data/selectcolumns>` widget.
```

Comment:
Perhaps, to be debated, but not for this pull request (would require changes elsewhere).